### PR TITLE
Cameras

### PR DESCRIPTION
**Pull request by StalkerSanya**
Adding many features: control, scripts, camera and maybe something else, I already don't remember.

### CODE REVIEW

File: controller/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(controller)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
```
Comment (l1va) [inline controller/CMakeLists.txt:7]: do you really need all these comments?

File: my_robot_gazebo/urdf/my_robot.urdf.xacro [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+
+<gazebo>
+  <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
+    <robotNamespace>/crobot</robotNamespace>
+  </plugin>
+</gazebo>
+
+<!-- camera -->
+  <gazebo reference="camera_link">
```
Comment (l1va) [inline my_robot_gazebo/urdf/my_robot.urdf.xacro:200]: might be better to put a camera in a separate of model file ?

File: scripts/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,202 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(scripts)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
```
Comment (l1va) [inline scripts/CMakeLists.txt:5]: have you seen some where cmakelists inside scripts folder? If you use scripts as package - bad package name then.

File: scripts/camerascript.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    print("Received an image!")
+    try:
+        # Convert your ROS Image message to OpenCV2
+        cv2_img = bridge.imgmsg_to_cv2(msg, "bgr8")
+    except CvBridgeError, e:
+        print(e)
+    else:
+    	global j
+    	if j < 6:
+    		filename = '/home/sanekcd/my_robot_ws/src/project1/screens_robot/camera_image' + str(j) + '.png'
```
Comment (l1va) [inline scripts/camerascript.py:23]: hardcode that will not work on my laptop

**Review by l1va — CHANGES_REQUESTED** [review]

File: controller/CMakeLists.txt (hunk 1 above)
Comment (StalkerSanya) [reply controller/CMakeLists.txt:7]: fixed

File: my_robot_gazebo/urdf/my_robot.urdf.xacro (hunk 2 above)
Comment (StalkerSanya) [reply my_robot_gazebo/urdf/my_robot.urdf.xacro:200]: fixed

File: scripts/CMakeLists.txt (hunk 3 above)
Comment (StalkerSanya) [reply scripts/CMakeLists.txt:5]: fixed

File: scripts/camerascript.py (hunk 4 above)
Comment (StalkerSanya) [reply scripts/camerascript.py:23]: fixed, but with filepath "~/Pictures/camera_image + str(j) + .png" it doesn't save images in end folder. I don't know why

**Pull request merged**